common/djangoapps/track/backends/tests/test_logger.py

```diff
@@ -47,6 +47,22 @@ def test_logger_backend(self):
         self.assertEqual(saved_events[0], unpacked_event)
         self.assertEqual(saved_events[1], unpacked_event)
 
+    def test_logger_backend_unicode_character(self):
+        """
+        When event information contain unicode characters
+        """
+        self.handler.reset()
+
+        event = {'string': '\xd3 \xe9 \xf1'}
```

Comment:
And if we are only testing translation of latin1, then we should confirm that extended unicode still breaks (have a negative test), and rename this to be latin1_character, not unicode_character.  But I'm thinking that the functionality should be broadened, offhand, to utf8, if we don't understand this better.